quarkus.ls/com.redhat.quarkus.ls/src/main/java/com/redhat/quarkus/services/QuarkusCompletions.java

```diff
@@ -209,5 +212,73 @@ private static String getPropertyName(String propertyName, boolean snippetsSuppo
 	private static void collectPropertyValueSuggestions(Node node, QuarkusProjectInfo projectInfo,
 			QuarkusCompletionSettings completionSettings, CompletionList list) {
 
+		if (node.getParent() != null &&
+			node.getParent().getNodeType() != Node.NodeType.PROPERTY) {
+			return;
+		}
+
+		PropertyKey key = getPropertyKeyFromProperty((Property) node.getParent());
+
+		if (key == null) {
+			return;
+		}
+
+		String keyText = key.getText().trim();
+
+		ExtendedConfigDescriptionBuildItem property = 
+			findBuildItemFromPropName(projectInfo.getProperties(), keyText);
+
+		Collection<String> enums = getEnums(property);
+
+		if (enums != null && enums.size() > 0) {
+			for (String e: enums) {
+				CompletionItem completionItem = new CompletionItem(e);
+				completionItem.setKind(CompletionItemKind.Value);

```

Comment:
Please use textedit (like completion on key). The text edit range should cover the whole property value. It will support the case that you have that:

`quarkus.log.console.async.overflow = B|L`

when you will apply completion, you should have `BLOCK` and not `BLOCKL`